mlir/utils/tuna/tuna-script.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+# For installing mysql 8.0 for testing, or for running with an isolated database.
+function mysql_setup_generic
+{
+    # Latest "generic" version.  (Must have libaio1 installed already.)
+    # Note that all this happens without privileges.
+    pushd /tmp
+    wget https://dev.mysql.com/get/Downloads/MySQL-8.0/mysql-8.0.34-linux-glibc2.28-x86_64.tar.gz
+    tar xf mysql-8.0.34-linux-glibc2.28-x86_64.tar.gz
+    mv mysql-8.0.34-linux-glibc2.28-x86_64 mysql
+    export PATH=$PATH:/tmp/mysql/bin
+    mysqld --initialize-insecure --datadir=/tmp/mysql/data
+    # --pid-file=/tmp/mysql/mysqld.pid --socket=/tmp/mysql/mysqld.sock
+    mysqld -D --basedir=/tmp/mysql --datadir=/tmp/mysql/data --log-error=/tmp/mysql/errors.log
+    popd
+
+    # Using this name should force socket access, which we want.
+    TUNA_DB_HOSTNAME=localhost
+    mysql --user root -e 'create database tuna;'
+}
+
+function tuna_setup
+{
+    startdir=`pwd`
+    rm -rf MITuna
+    git clone --branch pf-tuna-rocmlir-3 https://github.com/ROCmSoftwarePlatform/MITuna.git
+    cd MITuna
+    export TUNA_DIR=`pwd`
+
+    python3 -m venv myvenv
+    source myvenv/bin/activate
+
+    # --ignore-installed because of problems upgrading PyYAML.  See also -U.
+    python3 -m pip install -r requirements.txt --ignore-installed
+    python3 -m pip install scipy pandas
+
+    cd $startdir
+
+    export PYTHONPATH=$TUNA_DIR:$PYTHONPATH
+
+    if pgrep mysqld ; then
+        ${TUNA_DIR}/tuna/go_fish.py rocmlir --add_tables
+    fi
+}
+
+function clear_tables
+{
+    tablekind=$1
+
+    if [ "$tablekind" = "convolution" ]; then
```

Comment:
I was a little sloppy about naming.  The other ops only have one variation.

Reply:
`[[` here if possible

... heck, run `shellcheck` on this script just to be safe

Reply:
Re-raise nitpick / request for a `shellcheck`

Reply:
Ran shellcheck, and it didn't care about [ vs [[, though it did point out some useful places to double-quote and also urged $() over ``.